.changeset/light-news-compete.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ithaka/pharos': minor
```

Comment:
Would this qualify for a patch bump since I don't think we are adding any new functionality?